postgres/Dockerfile

```diff
@@ -2,21 +2,21 @@ FROM mcr.microsoft.com/windows/servercore:ltsc2016 AS download
 
 SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
 
-ENV PG_VERSION 10.2-1
+ENV PG_VERSION 11.7-1
 
-RUN Invoke-WebRequest $('https://get.enterprisedb.com/postgresql/postgresql-{0}-windows-x64-binaries.zip' -f $env:PG_VERSION) -OutFile 'postgres.zip' -UseBasicParsing ; \
+RUN Invoke-WebRequest $('http://get.enterprisedb.com/postgresql/postgresql-{0}-windows-x64-binaries.zip' -f $env:PG_VERSION) -OutFile 'postgres.zip' -UseBasicParsing ; \
```

Comment:
I still would prefer `https` here. Maybe only an issue with PowerShell and TLS 1.2.
But OK for now.